### ISSUE TITLE
clarify GO:0052083 suppression by symbiont of host cell-mediated immune response

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#18617]
can clarification be added to this term?

GO:0052083 suppression by symbiont of host cell-mediated immune response

Any process in which a symbiont stops, prevents, or reduces the frequency, rate or extent of the cell mediated immune response of the host organism. The host is defined as the larger of the organisms involved in a symbiotic interaction.

what does cell-mediated mean? Isn't everything cell-mediated?
Does it apply to a specific type of immune respone, or particular species?

**Comment by pgaudet** [comment]
I think this is the immune response that includes B cell and T cell processes. However I dont think we have a grouping term at that level. This is what we have under GO:0002376 immune system process: 

'activation of immune response'
'antigen processing and presentation'
'antigen sampling in mucosal-associated lymphoid tissue'
'B cell selection'
'hemocyte differentiation'
'hemocyte proliferation'
'immune effector process'
'immune response'
'immune system development'
'immunological memory process'
'leukocyte activation'
'leukocyte homeostasis'
'leukocyte migration'
'lymphocyte costimulation'
'myeloid cell homeostasis'
'production of molecular mediator of immune response'
'somatic diversification of immune receptors'
'T cell selection'
'tolerance induction' 

@addiehl ??

**Comment by ValWood** [comment]
The current term could be used by plants for example without qualification.
Is there a way to restrict it to animals? (definition and taxon restriction), so the plant people know to ignore this branch?

**Comment by addiehl** [comment]
This is an example of how the PAMGO folks relied on the elliptical language of the immunology community.  In typically, somewhat old style immunological language, cell mediated immunity (or immune response) really means T cell mediated immunity, and humoral immunity encompasses antibody responses and the B cells that make them (plus sometimes additional humoral factors). If you look at the synonyms of 'T cell mediated immunity' GO:0002456, you'll see I included cell mediated immunity as a synonym.

So if it were up to me, I would rename this as 'suppression by symbiont of host T cell-mediated immune response' and keep the original name as a synonym.

**Comment by ValWood** [comment]
That sounds good!

**Comment by ValWood** [comment]
@pgaudet could you add "Only in metazoa" to this one

**Comment by pgaudet** [comment]
Sorry @addiehl 
In fact this term has 3 children: 

'suppression by symbiont of host cell-mediated immune response'
- 'suppression by symbiont of host B-cell mediated immune response'
- 'suppression by symbiont of host T-cell mediated immune response'

**Comment by pgaudet** [comment]
So I cannot rename 'suppression by symbiont of host cell-mediated immune response' because of the B cell child. Note that the B cell term has no annotations.

**Comment by pgaudet** [comment]
> could you add "Only in metazoa" to this one

No ! this is 'by symbiont'. 
I can add never in pombe, cerevisiae, and Metazoa
OK ?

**Comment by ValWood** [comment]
Oh yes that won't help. IO was annotating plant pathogen not plant when I came to this. Then I had to look at all of the descendants to figure out if I was in the right branch.

**Comment by pgaudet** [comment]
So, no taxon constraint then ?

**Comment by ValWood** [comment]
I was looking for a collective word for the cell type and I found this

T cells (thymus cells) and B cells (bone marrow- or bursa-derived cells) are the major cellular components of the adaptive immune response. T cells are involved in cell-mediated immunity, whereas B cells are primarily responsible for humoral immunity (relating to antibodies).

So if you want to include both

suppression by symbiont of host **lymphocyte** cell-mediated** immune response

would be clear to curators (even if it isn't totally what the community would call it)
But I guess this term would never be used for direct annotation, it is only a grouping term, so that should be OK?

**Comment by ValWood** [comment]
you could add never in metazoa or plant.
We can't restrict to any microbes.

**Comment by pgaudet** [comment]
OK B cell is incorrect here. 

I can merge the T cell terms.

**Comment by addiehl** [comment]
In the main immune process hierarchy we have

leukocyte mediated immunity
--> lymphocyte mediated immunity
----> T cell mediated immunity (synonynym cellular mediated immunity)
----> B cell mediated immunity

In an ideal (Alex) world, the suppression terms would mirror these terms.

**Comment by pgaudet** [comment]
Those tickets are all really difficult ! 

GO:0046730 induction of host immune response by virus
- GO:0046737 induction of cell-mediated immune response in host by virus
- GO:0046736 induction of humoral immune response in host by virus
-  GO:0046738 induction of innate immune response in host by virus 

-> Need to be obsoleted, since the virus has to process that aims to stimulate the host immune response. 
No annotations. 
No slims, no mappings.

**Comment by addiehl** [comment]
I agree these virus terms are not appropriate, since it is the immune response that is the active process.

**Comment by ValWood** [comment]
Also
GO:0046730    induction of host immune response by virus
1 annotation

**Comment by pgaudet** [comment]
and the symbiont terms: 
- [ ] GO:0052559 | induction by symbiont of host immune response 
- [ ] GO:0052390 | induction by symbiont of host innate immune response - 0 annotations
- [ ]  GO:0036523 induction by symbiont of host cytokine secretion 1 annotation @RLovering

**Comment by pgaudet** [comment]
what about those: 


 - [ ] GO:0052104    induction by symbiont of host systemic acquired resistance
 - [ ]  GO:0052103    induction by symbiont of host induced systemic resistance

**Comment by pgaudet** [comment]
Some annotations to '[symbiont-mediated suppression of host cell-mediated immune response](https://amigo.geneontology.org/amigo/term/GO:0052083)' are to disruption of host macrophage processes; do we need a new term for this? 
- https://pubmed.ncbi.nlm.nih.gov/18474358/
- https://pubmed.ncbi.nlm.nih.gov/14557536/

**Comment by pgaudet** [comment]
2 CGD annotations to review: 
[PMID:19805528](https://amigo.geneontology.org/amigo/reference/PMID:19805528)
[PMID:17277107](https://amigo.geneontology.org/amigo/reference/PMID:17277107)

**Comment by pgaudet** [comment]
Obsolete: 
GO:0052155   perturbation of host cell-mediated immune response >> consider 'perturbation of host T-cell mediated immune response'
GO:0052083 symbiont-mediated suppression of host cell-mediated immune response >> consider symbiont-mediated suppression of host T-cell mediated immune response

**Comment by pgaudet** [comment]
GO:0052085 symbiont-mediated suppression of host T-cell mediated immune response
- [ ] change parent: GO:0039504 symbiont-mediated suppression of host adaptive immune response

**Comment by pgaudet** [comment]
All done
